### PR TITLE
Change Fluentd buffer files from /var/log to /var/run folder to avoid being treated as log files and getting rotated.

### PR DESCRIPTION
**Pull request by qingling128**

### CODE REVIEW

File: windows-installer/fluent-template.conf [hunk 1]
```diff
@@ -29,7 +29,7 @@
 <match **>
   @type google_cloud
   buffer_type file
-  buffer_path /var/log/google-fluentd/buffers
+  buffer_path /var/run/google-fluentd/buffers
```
Comment (igorpeshansky) [inline windows-installer/fluent-template.conf]: We probably should set it to something else, like `C:\temp\google-fluentd\buffers` or even `C:\ProgramData\google-fluentd\buffers`. In fact, we use `WIN_EVT_POS_FILE_PLACE_HOLDER` for the pos file, and substitute that at installation time. Why aren't we doing this for buffer files?

Comment (qingling128) [reply windows-installer/fluent-template.conf]: Good idea. Done.

**Comment by qingling128** [comment]
@igorpeshansky - Friendly ping. Note that this is a change into the `2.0.0` branch.

File: windows-installer/setup.nsi [hunk 2]
```diff
@@ -184,6 +184,8 @@ Section "Install"
 
   ; Create a directory to store position files.
   CreateDirectory ${MAIN_INSTDIR}\pos
+  ; Create a directory to store buffer files.
+  CreateDirectory ${MAIN_INSTDIR}\buffer
```
Comment (igorpeshansky) [inline windows-installer/setup.nsi]: `buffers`?

Comment (qingling128) [reply windows-installer/setup.nsi]: Sure. Done.

**Pull request merged**